mkdocs.yml

```diff
@@ -309,6 +288,27 @@ plugins:
       post_date_format: yyyy/MM/dd
       post_url_format: "{date}/{slug}"
       authors_file: "{blog}/.authors.yml"
+  - redirects:
+      redirect_maps:
+         jobs.md: https://jobs.applied-llms.org/
+         'hub/vertexai.md': 'examples/open_source.md#vertex-ai'
+         'hub/ollama.md': 'examples/ollama.md'
+         'hub/llama-cpp-python.md': 'examples/local_classification.md'
+         'hub/single_classification.md': 'examples/classification.md'
+         'hub/multiple_classification.md': 'examples/bulk_classification.md'
+         'hub/tables_from_vision.md': 'examples/extracting_tables.md'
+         'hub/pandas_df.md': 'examples/sqlmodel.md'
+         'hub/batch_classification_langsmith.md': 'examples/batch_job_oai.md'
+         'hub/action_items.md': 'examples/entity_resolution.md'
+         'hub/partial_streaming.md': 'concepts/partial.md'
+         'hub/extract_contact_info.md': 'examples/pii.md'
+         'hub/knowledge_graph.md': 'examples/knowledge_graph.md'
+         'hub/youtube_clips.md': 'examples/document_segmentation.md'
+         'hub/together.md': 'examples/open_source.md#mistral'
```

Comment:
make all the clients go to the integrations 

         'hub/anyscale.md': 'examples/open_source.md#anyscale'
         'hub/anthropic.md': 'examples/open_source.md#anthropic'
         'hub/cohere.md': 'examples/open_source.md#cohere'
         'hub/google.md': 'examples/open_source.md#google'
         
         
         integrations